doc/user/content/sql/system-catalog.md

```diff
@@ -434,7 +434,7 @@ Field       | Type       | Meaning
 `oid`       | [`oid`]    | A [PostgreSQL-compatible OID][oid] for the relation.
 `schema_id` | [`bigint`] | The ID of the schema to which the relation belongs.
 `name`      | [`text`]   | The name of the relation.
-`type`      | [`text`]   | The type of the relation: either `table`, `source`, or `view`.
+`type`      | [`text`]   | The type of the relation: either `table`, `source`, `view`, or `materialized view`.
```

Comment:
Should the description on line 428-429 be updated to include "materialized views" or is "view" enough for both?
> The `mz_relations` view contains a row for each table, source, and view in the
system.

Reply:
> or is "view" enough for both?

Good catch! It should be updated! Our docs should be consistent with PostgreSQL in treating "materializedview" as its own noun rather than a subtype of "view".